### DISCUSSION TITLE
Question about MIP start

### DISCUSSION THREAD

**Discussion by EnsignDaniels** [coin-or/Cbc#401] [Q&A]
Dear Colleagues,

I've been using Cbc in my own research and I believe I need some help. The question is, how can I assign the values (=0) to the variables, which aren't in the .sol file? To make it more clear, I have about 250000 variables in total, but only 850 of them are listed in the .sol, rest should be equal to zero. Is there a key in Cbc, that will allow it to set those rest variables as zeros?

Thanks

**Comment by jjhforrest** [comment]
Try using

-printing all -solution x.sol

to get sol file with 250,000 values

**Reply by EnsignDaniels** [reply]
Thanks for fast reply. But I believe this isn't exactly what I need. This feature provides all the data, including dual bounds etc. What I need is just all values of all variables in the model. Similar to one provided by Gurobi solver.
